core/gethwrappers/generation/compile_contracts.sh

```
@@ -3,7 +3,7 @@
 echo "compiling contracts"
 
 CDIR="$(dirname "$0")"
-COMPILE_COMMAND="$CDIR/../../contracts/scripts/native_solc_compile_all"
+COMPILE_COMMAND="$CDIR/../../../contracts/scripts/native_solc_compile_all"
```

interesting, not sure how it was working before 👀

---

think it just silently failed before was the issue

---

Yeah it only prints an error in the console but happily goes through with generation afterwards